ghost/milestones/test/Milestone.test.js

```diff
@@ -110,5 +112,15 @@ describe('Milestone', function () {
 
             assert(milestone.name === 'members-100');
         });
+
+        it('Will create event for new milestone', async function () {
```

Comment:
Might be worth adding explicit test that event isn't added for existing milestone - NABD if you don't wanna